src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.styled.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
```

Comment:
**suggestion** I recommend add here flex-wrap: wrap;, because I've noticed that loaders are in the same line with h3('nearest) element.
It looks like this in my local app.
![image](https://user-images.githubusercontent.com/107026487/185051997-37230a4f-ac58-46fa-b4de-06a6ca72d402.png)